### ISSUE TITLE
Migrate - Code of Conduct

### ISSUE THREAD

**Issue by bwklein** [thegooddocsproject/website-hugo#21]
_Originally posted by @bwklein in https://github.com/thegooddocsproject/website-hugo/issues/9#issuecomment-725749643_

From: https://thegooddocsproject.dev/CodeOfConduct

Maybe replaced with a link to: https://github.com/thegooddocsproject/governance/blob/master/CODE_OF_CONDUCT.md
Or somehow pull that content into the Hugo site through Hugo Modules.

Feedback from @barbaricyawps

**Comment by mgan59** [comment]
Wanted to point out, that @barbaricyawps is in-fight on a new version of the code-of-conduct.  This may or maynot be ready by time we get to this issue.  Recommend we do it last.

**Comment by mgan59** [comment]
> __From a Slack conversation regarding this work between Alyssa / Brian / Morgan__

The new version of the __Code of Conduct__ is in-flight in a Google-doc.  Once it is ready for review it will end up in the `governance` or `.github` repo.  We will be looking at how we can use a github-action to potentially sync or handle this further.

**Comment by barbaricyawps** [comment]
Status update on the Code of Conduct: The CoC committee has created two documents (the actual Code of Conduct and a second document about how to handle incident reports called Community Moderators). We have a motion open today to recommend these two documents to the Good Docs Project community. That motion is going to pass, so I'll probably submit these two documents (in Google Doc form) to the community for comment. We'll allow commenting for a week. If no blocking issues are raised, we'll adopt it.

**Comment by barbaricyawps** [comment]
Closing this issue because the CoC is merged now.

However, we might want to open a new issue because apparently the links on the main website are broken now.
